securedrop/tests/test_passphrases.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+
+from passphrases import PassphrasesGenerator, InvalidWordsListError
+
+# pylint: disable=unsupported-membership-test
+# False positive https://github.com/PyCQA/pylint/issues/3045
+
+
+class TestPassphrasesGenerator:
+    def test_default_generator(self):
+        # Given the default generator for the Securedrop app
+        generator = PassphrasesGenerator.get_default()
+        assert generator.available_languages == {"en"}
+
+        # When using it to generate a passphrase
+        # It succeeds
+        passphrase = generator.generate_passphrase()
+
+        # And a reasonably-secure passphrase was generated
+        assert passphrase
+        assert len(passphrase) > 16
+        assert len(passphrase.split(" ")) > 5
```

Comment:
These numbers don't match the default parameters in `passphrases.py`.

Reply:
Yeah I will update them